library/maven/artifacts.bzl

```diff
@@ -283,35 +283,35 @@ artifacts = {
     "org.hamcrest:hamcrest-core": "1.3",
     "org.hamcrest:hamcrest-library": "1.3",
     "org.javatuples:javatuples": "1.2",
-    "org.jetbrains.compose.compiler:compiler": "1.3.2",
-    "org.jetbrains.compose.animation:animation-core-desktop": "1.2.1",
-    "org.jetbrains.compose.desktop:desktop-jvm": "1.2.1",
-    "org.jetbrains.compose.foundation:foundation-desktop": "1.2.1",
-    "org.jetbrains.compose.foundation:foundation-layout-desktop": "1.2.1",
-    "org.jetbrains.compose.material:material-desktop": "1.2.1",
-    "org.jetbrains.compose.ui:ui-desktop": "1.2.1",
-    "org.jetbrains.compose.ui:ui-geometry-desktop": "1.2.1",
-    "org.jetbrains.compose.ui:ui-graphics-desktop": "1.2.1",
-    "org.jetbrains.compose.ui:ui-test-desktop": "1.2.1",
-    "org.jetbrains.compose.ui:ui-test-junit4-desktop": "1.2.1",
-    "org.jetbrains.compose.ui:ui-text-desktop": "1.2.1",
-    "org.jetbrains.compose.ui:ui-unit-desktop": "1.2.1",
-    "org.jetbrains.compose.runtime:runtime-desktop": "1.2.1",
-    "org.jetbrains.kotlin:kotlin-reflect": "1.7.20",
-    "org.jetbrains.kotlin:kotlin-stdlib-jdk7": "1.7.20",
-    "org.jetbrains.kotlin:kotlin-test": "1.7.20",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-core": "1.6.0",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm": "1.6.0",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-test": "1.6.0",
-    "org.jetbrains.kotlinx:kotlinx-serialization-core-jvm": "1.5.1",
-    "org.jetbrains.kotlinx:kotlinx-serialization-protobuf-jvm": "1.5.1",
-    # Find out the Skiko version we need by viewing dependencies of @maven//org_jetbrains_compose_ui_ui_graphics_desktop
```

Comment:
This was misleading, `bazel query 'deps(@maven//org_jetbrains_compose_ui_ui_graphics_desktop)'` reports whichever version is listed in this file, so cannot serve as a guide.